Sitka-Sound-Science-Center/sitka-landslide
# Add elevated risk window explainer

## Overview

Adds text to explain expanded risk window.  

Resolves #64 

### Demo
![demo-sitka-issue-64-desktop](https://user-images.githubusercontent.com/16727618/196546526-7919a2d6-3cca-4892-90b1-b1b25a431af5.png)

![demo-sitka-issue-64-mobile](https://user-images.githubusercontent.com/16727618/196546539-8b2a27bc-33f2-4c94-9a6f-827ac4dc7af5.png)

### Notes

This change renders a new `div` in the detail view for a 3 hr time period when the `riskIsElevatedFromPreviousPrecip` boolean flag (set up in #70) is true for that period.

This PR modified the `Icon` component because the SVGs added in this PR used 1) multiple `path` (versus all the previous svgs in the app which had 1-2) and 2) a `rect` element.  


## Testing Instructions

 * Load the site and look at [the Current risk](http://localhost:3008/detail/current/) view and the detail view for the forecast time periods. The rainfall total should move up and down. As progressing through time period if the `current` risk level declines from high or medium to a lower value, the subsequent 2 time periods should display the elevated risk window explainer. 

 * If progressing through the previously explained steps you find little variation in rainfall (which depends on the actual weather), run the data fetcher again with EXAGGERATION_FACTOR set, to be able to see some results with elevated risk, e.g. EXAGGERATION_FACTOR=5 yarn run get-data (run from frontend/. Adjust the number up or down, depending on the actual weather, to get it to produce a mix of risk levels). 

 * verify styles in desktop and mobile

## Checklist
